Examples/Example9.SendPacket/Example09.SendPacket.csproj

```diff
@@ -97,6 +97,10 @@
       <Project>{24262E52-1304-4A25-8F73-A3B06E40592E}</Project>
       <Name>SharpPcap</Name>
     </ProjectReference>
+     <Reference Include="PacketDotNet, Version=0.10.0.0, Culture=neutral, PublicKeyToken=null">

```

Comment:
The current version metadata in PacketDotNet.dll is indeed 0.13.0 but 
this is not currently reflected everywhere, e.g.:
https://github.com/chmorgan/sharppcap/blob/master/Examples/Example12.PacketManipulation/Example12.PacketManipulation.csproj#L64
I copied the version used in existing examples, regarding them as 
exemplary :)
Shall I make a sweep through each example's .csproj to update its 
PacketDotNet.dll reference to use version 0.13.0, and then test this by 
building the whole solution from fresh?

On 2016-07-09 00:05, chmorgan wrote:
> In Examples/Example9.SendPacket/Example09.SendPacket.csproj [1]:
> 
>> @@ -97,6 +97,10 @@
>> <Project>{24262E52-1304-4A25-8F73-A3B06E40592E}</Project>
>> <Name>SharpPcap</Name>
>> </ProjectReference>
>> + <Reference Include="PacketDotNet, Version=0.10.0.0,
> Culture=neutral, PublicKeyToken=null">
> 
> Didn't an earlier commit change the version to 0.13.0 from 0.10.0? But
> this one is 0.10.0?
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/chmorgan/sharppcap/pull/5#discussion_r70153244
> [2]
> https://github.com/chmorgan/sharppcap/pull/5/files/730d1c32d66bec0502d23bc7c1b53853dac63843#r70153244
> [3]
> https://github.com/notifications/unsubscribe/ABDTTuvXz3DqfgoFwXWLp_v__nB_dPZWks5qTtfBgaJpZM4ItpRH

-- 
http://www.cl.cam.ac.uk/~ns441/